integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -788,7 +788,7 @@ def test_div_overflow_exception_when_ansi(expr, ansi_enabled):
         assert_gpu_and_cpu_error(
             df_fun=lambda spark: _get_div_overflow_df(spark, expr).collect(),
             conf=ansi_conf,
-            error_message='java.lang.ArithmeticException: Overflow in integral divide')
+            error_message='ArithmeticException: Overflow in integral divide')
```

Comment:
Isn't the point to ensure we're throwing the same exception?  As such I'd rather see this test check for the correct exception class based on the Spark version -- if Spark 3.3 throws a different exception for this, then our plugin should throw the same class for the same type of exception on Spark 3.3.

I'm OK if we want to do this as a followup to refine the test if we aren't already doing the correct behavior now.  If we are, then this should choose a different message to check based on which Spark we're using as is done in other tests checking exception types.

Reply:
Yes, that's a good point. I filed https://github.com/NVIDIA/spark-rapids/issues/5196 as a follow-on.